README.md

```diff
@@ -77,7 +77,7 @@ Once the aboave has been completed, additional scripts will preformteh following
    for image storage.
 1. [Deploy
    CNV](https://github.com/openshift-kni/install-scripts/blob/master/CNV/deploy-cnv.sh). [Configure
-   a bridge on the `baremetal` interface on OpenShift nodes
+   a bridge on the `External` interface on OpenShift nodes
```

Comment:
On the final product will the interface be called External? If not, I'd say "a bridge on the external interface labeled `baremetal` on the OpenShift nodes."

Reply:
No the interface will no be called External. It is only name baremetal on the provision host.

We have 3 networks, Internal, External, and Storage.

So will change to - the interface on the Extermal network.

Reply:
OK.